sdcm/nemesis.py

```diff
@@ -186,6 +186,7 @@ class Nemesis:  # pylint: disable=too-many-instance-attributes,too-many-public-m
     has_steady_run: bool = False    # flag that signal that nemesis should be run with perf tests with steady run
     schema_changes: bool = False
     config_changes: bool = False
+    tablet_phase_1_supported = False  # flag that signal that the nemesis can run on a tablet-enabled cluster
```

Comment:
I think that a better approach would be to mark those that aren't supported.
Then, you can combine filters with ! and do things like "topology_changes" and "!tablets_supported" and more.

Reply:
Done